apps/dot-voting/app/components/VoteDetails/VotingResults.js

```diff
@@ -32,6 +32,7 @@ const VotingResults = ({ vote, voteWeights }) => {
           totalSupport={totalSupport}
           color={`${theme.accent}`}
           voteWeights={voteWeights}
+          displayYouBadge={true}
```

Comment:
We don't really need to add true when using boolean props, but I am not really opposed to being explicit about that if that helps you @PeterMPhillips , so I will not really suggest the change this time. Just tell me your opinion about this, to be consistent in the future.